extensions/template/README.md

```diff
@@ -2,7 +2,7 @@
 
 - **Title:** Template
 - **Identifier:**
-  <https://stac-extensions.github.io/template/v1.0.0/schema.json>
+  <https://stac-extensions.github.io/template/__STAC_VERSION__/schema.json>
```

Comment:
Shound'nt be https://linz.github.io/stac/**__STAC_VERSION__/template**/schema.json ?
I mean it does not match with what is in extensions/template/examples/collection.json